src/storage-preview/azext_storage_preview/operations/account.py

```diff
@@ -186,6 +194,10 @@ def remove_network_rule(cmd, client, resource_group_name, account_name, ip_addre
     if ip_address:
         rules.ip_rules = [x for x in rules.ip_rules if x.ip_address_or_range != ip_address]
 
+    if resource_id:
+        rules.resource_access_rules = [x for x in rules.resource_access_rules if
+                                       not (x.tenant_id == tenant_id and x.resource_id == resource_id)]
```

Comment:
```
rules.resource_access_rules = [r for r in rules.resource_access_rules if r.resource_id.lower() !=
                                       resource_id.lower() or r.tenant_id.lower() != tenant_id.lower()]
```
The above code has the logic of converting to lower case and then comparing, do we need to consider doing that here?